src/python/pants/init/engine_initializer.py

```diff
@@ -200,15 +201,17 @@ def run_console_rules(self, goals, target_roots):
     # Reduce to only applicable goals - with validation happening by way of `validate_goals()`.
     goals = [goal for goal in goals if goal in self.goal_map]
     subjects = self._determine_subjects(target_roots)
+    console = Console()
     # Console rule can only have one subject.
     assert len(subjects) == 1
     for goal in goals:
       try:
         goal_product = self.goal_map[goal]
+        params = Params(subjects[0], options_bootstrapper, console)
```

Comment:
In this case, the `Console` being a `Param` is:
1) more semantically correct, because in the case of `pantsd`, each run has a "new" console (we mutate global variables to accomplish this, but we wouldn't technically need to with appropriate use of injection like this)
2) it's easier to test, because we can inject a mock `Console` rather than using the mutation as above